content/posts/blog/2021-01-20.md

```diff
@@ -0,0 +1,55 @@
+---
+title: It is not a choice of the messenger
+date: 2021-01-20 06:30
+author: Edward Maurer
+blog_id: blog
+category: misc
+---
+
+## It is not a choice of the messenger
+
+Recently, several people reached out to me asking what kind of messenger they should be using now. Furthermore, they said that they actually do not understand what is going on and whether they should switch from one of the commonly known messengers to another one. 
+I was wondering how to answer this. Obviously, I could have stood-up and just advocated for XMPP (Extensible Messaging and Presence Protocol), but then I thought this alone is not a helpful answer. On the one hand, I see that many people just make a short-term decision. On the other hand, I also see they usually do not switch because they made a very evaluated choice for one specific messenger.
+
+Actually, most will just add another messenger to their list of messengers. Likely, that is more frustrating than helping, which brings me back to the beginning and the actual question: What issue or what problem does one want to solve here? What are the incentives? Can one decide towards a solution that, when the next hype chases through the internet, will have a better basis?
+
+A possible background of that question may be different expectations of some functionality, the basic requirement of effective data privacy and sovereignty or simply the possibility to reach all my contacts. However, that change will still require leaving behind the previous messaging system and switching to a new one, which almost always results in the loss of the existing contact connections. Furthermore, in terms of privacy, one is almost always required to provide a phone number. Again, what does one will try to solve?
+
+### Sovereignty of Communication
+
+So, as you likely already wondered, why am I going to advocate for XMPP now? I think the choice one should do firsthand, is not a choice of the messenger you use, but of the underlying software technology. Obviously, one should actually always do that. So, here we are: Consider the choice of technology first before you jump from one recommendation to the next one.
+
+> XMPP is an open protocol, as HTTP is for websites. It mostly does not matter how your website looks like, but everyone can interact with it. This principle is the idea behind XMPP for instant messaging.
+
+We at the XMPP Standards Foundation, but also many others involved in this open technology, think that XMPP is a good choice as a communication technology when it comes to this question. XMPP as a protocol has been around for about two decades since its creation in 1999 and a lot of experience has been gathered during that period. Mainly, this includes several important key indicators for the choice of technology and enabling of sovereignty as 
```

Comment:
Remove "when it comes to this question" and end the sentence after "technology". We think it's good in general, not just with respect to data privacy :)

You mention data sovereignty here again, but it wasn't really ever defined. Maybe define it the first time you mention it? I can take a guess, but I'm not really sure what that means.